### ISSUE TITLE
Fix users query

### ISSUE THREAD

**Issue by schnie** [astronomer/houston-api#77]
The `users` query is too loose now. Details to come.

**Comment by petedejoy** [comment]
A few notes from our chat:
 - Refactor current users query to be a systemUsers query. Add system admin permissions here.
 - Add workspaceUsers query that requires workspaceUuid as an arg. Add workpsace admin permissions here.

**Comment by ryw** [comment]
How is this going @petedejoy?

**Comment by petedejoy** [comment]
Should have a pair of PRs ready for review today! Got sucked into a million other things the last 2 weeks

**Comment by petedejoy** [comment]
Houston PR: https://github.com/astronomer/houston-api/pull/95
Orbit PR: https://github.com/astronomer/orbit-ui/pull/124

Will work on writing a unit test for my new Houston resolver today as well.

**Comment by petedejoy** [comment]
@schnie - just pushed up the changes we talked about this afternoon. I added a third `workspaceUser` query that takes workspaceUuid and username as required args. It basically just pulls some of the functional stuff we were doing to Houston. I'm pretty stoked about how it turned out- it's much cleaner and aligns super nicely with the "a resolver for everything" GraphQL philosophy. I also have unit tests in there for all three of the new resolvers.
